pkg/interface/src/views/components/MentionText.tsx

```diff
@@ -53,10 +54,11 @@ export function Mention(props: {
   }, [showOverlay]);
 
   return (
-    <Box position='relative' display='inline-block' cursor='pointer'>
+    <Box position='relative' display='inline-block' cursor='pointer' {...rest}>
       <Text
         onClick={() => toggleOverlay()}
-        mx={1}
+        marginLeft={first? 0 : 1}
```

Comment:
I don't know if the ternary should be right up in there, but it does seem to work.